source/common/upstream/health_discovery_service.cc

```diff
@@ -22,6 +27,7 @@ void HdsDelegate::setRetryTimer() {
 }
 
 void HdsDelegate::establishNewStream() {
+
```

Comment:
Nit: prefer not to add unnecessary extra whitespace.

Reply:
Done